backend/src/app.ts

```diff
@@ -48,4 +51,35 @@ app.get('/reviews/:idType/:id', async (req, res) => {
   res.status(200).send(JSON.stringify(reviews));
 });
 
+app.post('/landlords', async (req, res) => {
+  try {
+    const doc = landlordCollection.doc();
+    const landlord: Landlord = req.body as Landlord;
+    doc.set(landlord);
+    res.status(201).send(doc.id);
+  } catch (err) {
+    res.status(400).send(err);
+  }
+});
+
+app.get('/reviews/', async (req, res) => {
```

Comment:
Remove the trailing `/` at the end of the URI since we just need to call `reviews?q=something`